common/ConfigUtil.cpp

```diff
@@ -10,18 +10,17 @@
  */
 
 #include <config.h>
```

Comment:
Please leave a blank line after `<config.h>` to avoid having it sorted by tools. `<config.h>` must be included first in `.cpp` files.